flutter/test/integrations/on_error_integration_no_binding_test.dart

```diff
@@ -1,59 +0,0 @@
-import 'package:flutter_test/flutter_test.dart';
-import 'package:mockito/mockito.dart';
-import 'package:sentry/sentry.dart';
-import 'package:sentry_flutter/src/integrations/on_error_integration.dart';
-import 'package:sentry_flutter/src/sentry_flutter_options.dart';
-
-import '../mocks.dart';
-import '../mocks.mocks.dart';
-import 'mock_platform_dispatcher.dart';
-
-void main() {
```

Comment:
Same as https://github.com/getsentry/sentry-dart/pull/1184/files#r1060373844

Reply:
I re-added it